.changeset/strange-nails-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@compiled/codemods': minor
+---
+
+add support for keyframes import with alphabetical sorting
```

Comment:
Can you capitalize this from `add` to `Add`

Reply:
Done